binary.js

```diff
@@ -4,18 +4,46 @@ var testArrayGenap = [40, 18, 22, 32, 90, 10, 10, 22, 8]
 var testArrayGanjil = [3, 31, 89, 53, 53, 85, 77, 21, 55]
 
 function ownSort(arr) {
-  // Your sorting code
+  var sorted = 0
+  for (var i = 1; i < arr.length; i++) {
+    sorted = i
```

Comment:
```suggestion
    let sorted = i
```

deklarasi variable yang di line 7 bisa dipindah kesini saja